dockerfiles/models-gpu/Dockerfile

```diff
@@ -76,6 +68,8 @@ RUN mkdir -p /var/run/sshd && \
     echo "    StrictHostKeyChecking no" >> /etc/ssh/ssh_config.new && \
     mv /etc/ssh/ssh_config.new /etc/ssh/ssh_config
 
+ARG OMPI=4.0.3
```

Comment:
suggest to separate these kind of docker images to class A + B where A is mlrun oriented where B is orthogonal. That way, you build this docker image (pretty much) once, push it, and reuse it. unless @yjb-ds would say it might change between every version and is tightly coupled with mlrun.

Reply:
That is something we need to consider more thoroughly w.r.t. to how this OMPI thing works, and anyways if we'll do it, we'll do it in a separate PR